src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -254,13 +247,15 @@ ov::AnyMap get_baseline_common_config() {
 
 ov::AnyMap get_default_common_config(const std::shared_ptr<ov::Model>& model) {
     auto config = get_baseline_common_config();
+    if (is_cw_compressed(model)) {
+        config.emplace("NPUW_SLICE_OUT", "YES");
+    }
```

Comment:
SLICE_OUT can be enabled unconditionally, why only CW?

Reply:
Thought there were regressions for this case, will change

Reply:
Done